sdk/quantum/quantum-jobs/package.json

```diff
@@ -6,79 +6,117 @@
   "dependencies": {
     "@azure/core-paging": "^1.1.1",
     "@azure/core-http": "^1.2.0",
-    "tslib": "^2.0.0"
+    "tslib": "^2.0.0",
+    "@opentelemetry/api": "^0.10.2"
   },
   "keywords": [
     "node",
     "azure",
     "typescript",
     "browser",
-    "isomorphic"
+    "isomorphic",
+    "cloud"
   ],
   "license": "MIT",
-  "main": "./dist/quantum-jobs.js",
-  "module": "./esm/index.js",
-  "types": "./esm/index.d.ts",
+  "main": "dist/index.js",
+  "module": "./dist-esm/src/index.js",
+  "types": "./types/src/index.d.ts",
```

Comment:
Usually, we provide the trimmed-down single-file version of the types with the file name made up of the package name such as...
`"./types/latest/quantum-jobs.d.ts"`

Reply:
This trimmed version is the output of `api-extractor`, we need to provide the file name at `api-extractor.json`.

Reply:
This part is new to me. I will schedule a call with you to discuss the details. For now, changed it